src/main/java/com/google/sps/servlets/AnswerQuestionServlet.java

```diff
@@ -35,7 +35,23 @@
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
 
     String gameInstance = request.getParameter("gameInstance");
-    String student = request.getParameter("student");
+
+    if(gameInstance==null || gameInstance.isEmpty()){
+        response.setStatus(500);
+        response.getWriter().println("Room not specified");
+        return;
+    }
+
+    String token = request.getParameter("student");
+    if(token==null || token.isEmpty()){
+        response.setStatus(500);
+        response.getWriter().println("Room not specified");
```

Comment:
not that it matters a lot, but the response should say something like: user not logged in right? Because the ID token would be empty?

Reply:
You are right thank you!